### PR TITLE
Add legacy feedback

### PR DESCRIPTION
**Pull request by mcbouslog**
Package: lib-classifier

Towards #205, see #443. 

Describe your changes:
- adds legacy feedback files from PFE
    - PFE features/feedback/classifier => FEM components/Classifier/components/Feedback
    - PFE features/feedback/shared/helpers => FEM store/feedback/helpers
    - PFE features/feedback/shared/strategies => FEM store/feedback/strategies

I think is the most basic initial add, therefore there'll certainly need to be clean up, but any subsequent PRs related to feedback should only be a few files and changes specific to FEM feedback implementation.

The next feedback related PR will be to add a minimal feedback store, then basic UI.

I'm thinking it might be helpful to add a specific issue to review and clean up feedback files after basic implementation.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
I don’t think the React components are needed, because we’ll be writing a new feedback component using Grommet.

File: packages/lib-classifier/src/store/feedback/helpers/generate-rules.spec.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+// import { expect } from 'chai';
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/feedback/helpers/generate-rules.spec.js]: Why are these tests completely commented out? 🤔

Comment (eatyourgreens) [reply packages/lib-classifier/src/store/feedback/helpers/generate-rules.spec.js]: They aren't commented out in PFE, but there are no  notes here to say why they've been removed.

**Comment by eatyourgreens** [comment]
I've removed references to the lab components, uncommented the commented-out tests and linted to convert the new code to StandardJS.

**Comment by mcbouslog** [comment]
This LGTM, thank you @eatyourgreens.

This takes care of a little cleanup that was included in #533, but better to include here.

File: packages/lib-classifier/src/store/feedback/helpers/generate-rules.spec.js (hunk 1 above)
Comment (mcbouslog) [reply packages/lib-classifier/src/store/feedback/helpers/generate-rules.spec.js]: Intended to fix, lint, etc. in #533, but better to include in this PR, thanks!

File: packages/lib-classifier/src/store/feedback/helpers/generate-rules.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import _ from 'lodash'
```
Comment (mcbouslog) [inline packages/lib-classifier/src/store/feedback/helpers/generate-rules.js]: Should all `lodash` imports be scoped to just the methods used in the file?

File: packages/lib-classifier/src/store/feedback/strategies/drawing/column/feedback-mark.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
```
Comment (mcbouslog) [inline packages/lib-classifier/src/store/feedback/strategies/drawing/column/feedback-mark.js]: Should PropTypes be scoped to just the types used in the file?

File: packages/lib-classifier/src/store/feedback/helpers/generate-rules.js (hunk 2 above)
Comment (eatyourgreens) [reply packages/lib-classifier/src/store/feedback/helpers/generate-rules.js]: If these helpers are acting on MST nodes, we should look at scrapping lodash in favour of using MST's methods for maps and iterators.

**Comment by eatyourgreens** [comment]
I'm going to fix the merge conflict, then merge this.

**Pull request merged**